### PR TITLE
docs(zeebe-exporters): add form index infos

### PR DESCRIPTION
**Pull request by nicpuppa**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md
Comment (koevskinikola) [inline docs/self-managed/zeebe-deployment/exporters/opensearch-exporter.md:1]: :x: We should also add it in the `#Example` section under [line 225](https://github.com/camunda/camunda-docs/pull/2926/files#diff-59c64515c5389ca3718fb1e24c662e9abc8f003e44f1d119c09cd57b67a93299R225).

File: versioned_docs/version-8.3/self-managed/zeebe-deployment/exporters/opensearch-exporter.md
Comment (koevskinikola) [inline versioned_docs/version-8.3/self-managed/zeebe-deployment/exporters/opensearch-exporter.md:1]: :x: We should also add it in the `#Example` section under [line 199](https://github.com/camunda/camunda-docs/pull/2926/files#diff-b19ea2ec401fc96f54d0aee8c5a1bdfc56d11873830bbd2c1da7b64b4445877aR199).

File: docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md
Comment (koevskinikola) [inline docs/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md:1]: :x: We should also include the `form` flag in the `#Example` section.

File: versioned_docs/version-8.3/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md
Comment (koevskinikola) [inline versioned_docs/version-8.3/self-managed/zeebe-deployment/exporters/elasticsearch-exporter.md:1]: :x: We should also include the `form` flag in the `#Example` section.

**Review by koevskinikola — CHANGES_REQUESTED** [review]
Thanks for adding this @nicpuppa.

I think we should also include the `form` flag in the `#Example` section of each of these documents.

**Comment by nicpuppa** [comment]
Thank for the suggestion @koevskinikola

**Review by koevskinikola — APPROVED** [review]
LGTM :100:

**Review by christinaausley — APPROVED** [review]

**Pull request merged**